src/ceph-daemon/ceph-daemon

```diff
@@ -226,6 +226,21 @@ def is_fsid(s):
         return False
     return True
 
+def infer_fsid():
```

Comment:
There is also a way to infer the fsid by looking as the ceph-volume LVs. 

```suggestion
def infer_fsid_from_dirname():
```

?

Reply:
maybe if c-v weren't so slow...